content/docs/connect/connect-from-any-app.md

```diff
@@ -2,42 +2,29 @@
 title: Connect from any application
 subtitle: Learn how to connect to Neon from any application
 enableTableOfContents: true
-updatedOn: '2023-09-15T13:00:43Z'
+updatedOn: '2023-10-05T21:03:44.956Z'
 ---
-When connecting to Neon from an application or client, you connect to a database in your Neon project. In Neon, a database belongs to a branch, which may be the primary branch of your project (`main`) or a child branch. The primary branch in a Neon project is created with a ready-to-use database named `neondb`.
+When connecting to Neon from an application or client, you connect to a database in your Neon project. In Neon, a database belongs to a branch, which may be the primary branch of your project (`main`) or a child branch.
 
-To connect to a database, you must connect to the branch where the database resides, and you must do so by connecting through a compute endpoint associated with the branch.
-
-```text
-Project
-    |----primary branch (main) ---- compute endpoint <--- application/client
-             |    |
-             |    |---- database (neondb)
-             |
-             ---- child branch ---- compute endpoint <--- application/client
-                            |
-                            |---- database (mydb)  
-```
-
-You can obtain the connection details you require from the **Connection Details** widget on the **Neon Dashboard**. Select a branch, a role, and the database you want to connect to. A connection string is constructed for you.
+You can obtain the database connection details you require from the **Connection Details** widget on the **Neon Dashboard**. Select a branch, a compute, a database, and a role. A connection string is constructed for you.
```

Comment:
A few points here:

- This is a style thing. Maybe use a simpler word than "obtain": find, or get. 
- See if there is a way of avoiding saying 'connection details' twice in the same sentence. 
- Is the Connections Details area in the dashboard really a widget? I think of widgets as user-configurable: user can select which widgets to display in your dashboard, the behavior/info for each widget, that kind of thing.

Here's an attempt:

"From the **Connection Details** panel in the **Neon Dashboard**, you can find the details you'll need to connect to any database associated with your project. "